notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
```

Comment:
<p>no first person. Replace we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
But in the template, it's using we everywhere.

Reply:
Reference: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb